toolchest_client/tools/api.py

```diff
@@ -124,17 +151,17 @@ def test(inputs, output_path, tool_args=""):
 
     instance = Test(
         tool_args=tool_args,
-        output_name='output.txt',
+        output_name='Aligned.out.sam',
```

Comment:
Was this intentional?

Reply:
Yes. For now, we don't have control over the output names of STAR, and before we implement multiple output files + compressing multiple files into one outputted zip, we should just output the `.sam` file.

Reply:
This is the `Test` instance, not `STAR`

Reply:
Should have specified what I was commenting on, whoops

Reply:
My bad -- should be fixed now